test/ProgressOnderwijsUtilsTests/TransactedLocalConnection.cs

```diff
@@ -14,15 +14,10 @@ public class TransactedLocalConnection : IDisposable
         public TransactedLocalConnection()
         {
             Transaction = new CommittableTransaction();
-            Context = new SqlCommandCreationContext(new SqlConnection(Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? @"Server = (localdb)\MSSQLLocalDB; Integrated Security = true"), 60, SqlCommandTracer.CreateAlwaysOffTracer());
+            Context = new SqlCommandCreationContext(new SqlConnection(Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? @"Server = localhost; Integrated Security = true"), 60, SqlCommandTracer.CreateAlwaysOffTracer());

```

Comment:
Waarom localhost?  De intentie is dat dit kan draaien zonder een volledige  ms sql server install.